src/k8s-configuration/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
Remove these 2 lines as we dropped support for Python 2.

Reply:
Removed